.github/workflows/dependabot_automerge.yml

```diff
@@ -0,0 +1,31 @@
+name: "Dependabot Automerge"
+
+on:
+  workflow_run:
+    workflows: ["CI Build"]
+    types:
+      - completed
+
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+
+    if: ${{ github.event.workflow_run.conclusion == 'success' && github.actor == 'dependabot[bot]' }}
+    steps:
+      - uses: actions/github-script@v5
+        env:
+          pull_request_number: ${{ toJSON(github.event.workflow_run.pull_requests[0].number) }}
+        with:
+          script: |
+            github.pulls.createReview({
```

Comment:
Should we use `github.rest.pulls.*` if using `actions/github-script@v5`? https://github.com/actions/github-script#breaking-changes-in-v5

Reply:
Thanks